client/src/views/ScorecardView.vue

```diff
@@ -12,14 +12,14 @@
       <PredictionPanel />
 
       <!-- Only display the side-panel, full-width on mobile. -->
-      <ScorecardMapViewBar class='is-hidden-mobile' />
-      <div class='columns is-variable is-centered is-gapless'>
+      <ScorecardMapViewBar />
```

Comment:
How does this look on mobile with pills?

Reply:
The pills appear on desktop and disappear on mobile 
![image](https://user-images.githubusercontent.com/7783393/193075230-10aae34e-5dca-41b7-b443-0488d3822cf8.png)